recipes/sarpy/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "sarpy" %}
+{% set version = "1.2.41" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sarpy-{{ version }}.tar.gz
+  sha256: 01539cc36271d435bb982917684bdc726850578c6cacccecbe59bf708a1515c2
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.11.0
+    - python >=3.6
+    - scipy
+
+test:
+  imports:
+    - sarpy
+    - sarpy.annotation
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ngageoint/sarpy
+  summary: Python tools for reading, writing, and simple processing of complex SAR data and other associated data.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Update to the one in the tarball:
```suggestion
  license_file: LICENSE
```